src/gpu/generic/convolution_deconvolution.hpp

```diff
@@ -0,0 +1,381 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef GPU_GENERIC_CONVOLUTION_DECONVOLUTION_HPP
+#define GPU_GENERIC_CONVOLUTION_DECONVOLUTION_HPP
+
+#include "common/c_types_map.hpp"
+#include "common/primitive.hpp"
+#include "common/type_helpers.hpp"
+#include "common/utils.hpp"
+#include "gpu/gpu_deconvolution_pd.hpp"
+#include "gpu/intel/compute/utils.hpp"
+#include "gpu/intel/gpu_primitive.hpp"
+#include "gpu/intel/primitive_conf.hpp"
```

Comment:
The generic implementations cannot include anything from vendor specific locations.

Reply:
Fixed